### DISCUSSION TITLE
Welcome to i18next Discussions!

### DISCUSSION THREAD

**Discussion by jamuhl** [i18next/i18next#1524] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Opus4210** [comment]
I have a one-liner implementation of i18next that is marked as 
`// i18next, v1.7.7`
`// Copyright (c)2014 Jan Mühlemann (jamuhl).`

The code itself is very long, so I won't paste it here.

My problem with it is that it contains a cookie that doesn't meet Chrome's SameSite attribute requirement, If I understood the minified code better, I would just add `SameSite=Lax` to it, but I'm not grokking it. Can someone help me with this change?

**Reply by jamuhl** [reply]
v1.7.7 😲 -> it's more than time to update that dependency.

this is not a discussion - this is not even an issue - I understand you got a problem but searching a code line in a minified file that is totally outdated is not really an issue in the current module.

Good luck anyway...

**Reply by Opus4210** [reply]
Fair enough. I wasn't sure where to start, since this one line of code is my first exposure to this module. More to the point, then: if I include a script tag with one of the CDNs mentioned in Getting Started, will that script reference replace this line of code?

**Reply by jamuhl** [reply]
Not that easy. V19.x.x is completely different -> you will need to add plugins as needed and set configuration correctly. Also if you're taking about an integration using jquery -> use jquery-i18next.
